plugin/hover.py

```diff
@@ -122,6 +123,13 @@ def request_symbol_hover_async(self, listener: AbstractViewListener, point: int)
         hover_promises = []  # type: List[Promise[ResolvedHover]]
         document_position = text_document_position_params(self.view, point)
         for session in listener.sessions_async('hoverProvider'):
+            range_hover_provider = session.get_capability('experimental.rangeHoverProvider')
+            if range_hover_provider:
+                region = first_selection_region(self.view)
+                if region is not None:
+                    if region.contains(point):
+                        # hovering selection or only text was selected
+                        document_position = text_document_range_params(self.view, region)
```

Comment:
This is not correct. `document_position` will be set here and then we'll send it to other sessions also on next loop iterations.

Reply:
@rchl you prefer [this](https://github.com/sublimelsp/LSP/pull/1900/commits/7e5e0450e2aed35ca8d71297f58934e281af2ae5) or override the variable ?